jcli/src/jcli_app/vote/tally/mod.rs

```diff
@@ -12,18 +12,34 @@ pub enum Tally {
     /// The decryption share data will be printed in hexadecimal encoding
     /// on standard output.
     DecryptionShare(decryption_tally::TallyGenerateDecryptionShare),
+    /// Create a decryption share for private voting tally.
+    ///
+    /// The decryption share data will be printed in hexadecimal encoding
+    /// on standard output.
+    VotePlanDecryptionShares(decryption_tally::TallyGenerateVotePlanDecryptionShares),
```

Comment:
If we're going to deprecate `decryption-share` above, just name the subcommand `decryption-shares`.
Nobody needs the carpal tunnel syndrome typing these commands.